### ISSUE TITLE
Add Bread Crumbs to Navbar

### ISSUE THREAD

**Issue by maxdumas** [TechAtNYU/intranet#93]
We want to it so that when we are `/r/events/edit/541f60e5ff7ba819334c8ffa`, our bread crumb in the navbar should be `Tech@NYU Intranet > Event > Edit > 541f0...`, or maybe `Tech@NYU Intranet > Event > Edit > Tech@NYU After Hours #3` once we get #71 implemented.

**Comment by maxdumas** [comment]
@ethanresnick @grungerabbit I was thinking, is it worth it to do this before we really sit down and think about how we want to structure the interface of the intranet? I feel like something like this is really dependent on that. I'll still look into it, because it's definitely another big UX boost in the interim, but it's something to think about!

**Comment by ethanresnick** [comment]
You're totally right. It's probably not what we'll end up with long term, so it'd only be useful as an interim fix. If you have time to implement it, feel free. But because it may go, I'd say its not a huge priority
